package.json

```diff
@@ -43,9 +43,9 @@
     "webpack-merge": "^4.1.4"
   },
   "dependencies": {
-    "@material-ui/core": "^3.9.0",
+    "@material-ui/core": "^4.0.0-rc.0",
     "@material-ui/icons": "^2.0.3",
-    "@material-ui/styles": "^3.0.0-alpha.10",
+    "@material-ui/styles": "^4.0.0-rc.0",
```

Comment:
Same with this, should be `4.1.2` (latest at time of writing)

Reply:
Done